.github/workflows/package.yml

```diff
@@ -266,7 +266,7 @@ jobs:
   visual-studio:
     strategy:
       matrix:
-        os: [ windows-2019, windows-2022 ]
+        os: [ windows-2025, windows-2022 ]
```

Comment:
ditto: no need to run on both versions of Windows

Reply:
Until the latest version we published both VS2019 and VS2022 packages. Now I guess we can't publish VS2019 anymore...